src/v0/destinations/fb/transform.js

```diff
@@ -103,10 +103,15 @@ function sanityCheckPayloadForTypesAndModifications(updatedEvent) {
         break;
       case 'ud[ge]':
         if (clonedUpdatedEvent[prop] && clonedUpdatedEvent[prop] !== '') {
-          isUDSet = true;
-          clonedUpdatedEvent[prop] = sha256(
-            clonedUpdatedEvent[prop].toLowerCase() === 'female' ? 'f' : 'm',
-          );
+          if (typeof clonedUpdatedEvent[prop] !== 'string') {
```

Comment:
@anantjain45823 We seem to be only considering strings here, are booleans also not supported?

Reply:
No, this field is taken from `gender` field which should be a string